### PR TITLE
Docs for Jupyter Scheduler integration

### PR DESCRIPTION
**Pull request by iameskild**
## Reference Issues or PRs

These are docs for the new Jupyter-Scheduler integration, see https://github.com/nebari-dev/nebari/pull/1832

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [x] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [x] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [x] This content adheres to the Nebari style guides.

#### Non-text content

- [x] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (pavithraes) [inline yarn.lock:1]: Do we need to add this file to the repo?

File: docs/docs/tutorials/run-notebooks-as-a-job.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+3. Navigate to `~/.local/share/jupyter/scheduler_staging_area`
+4. Find the folder that corresponds to the job ID that failed
+5. Open (or `cat`) the `output.ipynb` to view the notebook as it was run
+
+
+:::info
+Notebook job details can also be viewed from the `<nebari-domiain>/argo` UI.
+:::
+
+Lastly, if the job fails without writing to this `scheduler_staging_area`, or the job status is stuck in `In progress` mode for an extended period of time, have an administrator try and view the specific the logs on the user`s JupyterLab server pod or on the workflow pod itself.
```
Comment (pavithraes) [inline docs/docs/tutorials/run-notebooks-as-a-job.md:127]:
```suggestion
Lastly, if the job fails without writing to this `scheduler_staging_area`, or the job status is stuck in `In progress` mode for an extended period of time, have an administrator try and view the specific logs on the user's JupyterLab server pod or on the workflow pod itself.
```

**Review by pavithraes — APPROVED** [review]
Thanks, @iameskild! Just have two minor comments and everything else looks great, so approving. Please feel free self-merge when ready :)

File: yarn.lock (hunk 1 above)
Comment (iameskild) [reply yarn.lock]: I'm surprised this wasn't caught by the gitignore... either way good catch!

**Comment by pavithraes** [comment]
This PR will be merged with https://github.com/nebari-dev/nebari/pull/1832

**Pull request merged**